inference-engine/include/vpu/myriad_plugin_config.hpp

```diff
@@ -52,6 +52,21 @@ DECLARE_VPU_MYRIAD_CONFIG_KEY(PLATFORM);
 DECLARE_VPU_MYRIAD_CONFIG_VALUE(2450);
 DECLARE_VPU_MYRIAD_CONFIG_VALUE(2480);
 
+/**
+ * @brief This option allows to specify device memory type.
+ */
+DECLARE_VPU_MYRIAD_CONFIG_KEY(MOVIDIUS_DDR_TYPE);
+
+/**
+ * @brief Supported keys definition for VPU_MYRIAD_CONFIG_KEY(MOVIDIUS_DDR_TYPE) option.
+ */
+DECLARE_VPU_MYRIAD_CONFIG_VALUE(NOT_PROG);
```

Comment:
In our case values must be MICRON_1GB, MICRON_2GB, HYNIX_2GB, SAMSUNG_2GB. 
Now key values specifies package type, not DDR type.

Reply:
Oh, I see. For some reason I was sure that I need to change the code of the main method in ddrInit.